web-app-nginx.conf

```diff
@@ -23,8 +24,45 @@ server {
         try_files $uri /index.html;
         add_header Last-Modified $date_gmt;
         add_header Cache-Control 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
+        add_header Access-Control-Allow-Origin *;
         if_modified_since off;
         expires off;
         etag off;
     }
+
+    location /api/list {
+        proxy_pass http://web-svc.emojivoto:8080;
```

Comment:
Im not sure this is correct. @maxlegault can you confirm?

Reply:
Yeah I'm not sure why this required either. Don't we expose the `web-svc` as a mapping so that requests to `/api/*` are routed to it?

Reply:
Yes. Ambassador is taking care of the ingress for our demo clusters. @arturogonzalez58 please just leave line 12 of this file as is. Every other addition in this file can be removed.